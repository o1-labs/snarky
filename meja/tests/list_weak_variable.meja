let x = [];

let y = true :: x;

let z = 1 :: x;
