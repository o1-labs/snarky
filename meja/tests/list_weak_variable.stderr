File "tests/list_weak_variable.meja", line 5, characters 13-14:
Error: Incompatible types list(int) and list(bool):
       Cannot unify int and bool
